Framework/PythonInterface/test/python/mantid/plots/plotfunctionsTest.py

```diff
@@ -162,6 +159,14 @@ def test_cannot_make_waterfall_plot_with_one_line(self):
 
         self.assertFalse(ax.is_waterfall())
 
+    def test_overplotting_supports_first_time_plot(self):
+        # Note how we call overplot true first time round
+        for _ in range(2):
+            plot([self._test_ws], wksp_indices=[1], overplot=True)
+        ax = plt.gca()
+        line_colors = [line.get_color() for line in ax.get_lines()]
+        self.assertNotEqual(line_colors[0], line_colors[1])
```

Comment:
Why are you checking whether the two lines have the same colour?  Shouldn't this just test that only one figure has been created with both lines on it? If you want to make sure the lines are distinct you could just compare them directly rather than their colour.

Reply:
Copy + Paste from the test above as a template, good suggestions I'll fix that up now